tests/queries/0_stateless/02932_parallel_replicas_fuzzer.sql

```diff
@@ -0,0 +1,38 @@
+SET parallel_replicas_for_non_replicated_merge_tree=1;
```

Comment:
Do we have a similar test for JOINs? Also, the test doesn't cover custom key queries

Reply:
JOINSs are supported with the old infrastructure but not supported in the analyzer. The tests are either running with the analyzer disabled explicitly or under analyzer_tech_debt.txt

The first test comes from the fuzzer. It doesn't cover custom key queries because there is no sane way to use them in that scenario.

The second query is to test FINAL, which is enabled for custom key parallel replicas.